.github/lint-requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django~=2.2.6
+pylint-django==2.0.11
```

Comment:
This file is for `pylint` to install, as it doesn't need all of the dependencies in `requirements.txt`